### DISCUSSION TITLE
How can I reset my Radio's root password ?

### DISCUSSION THREAD

**Discussion by faxm0dem** [LMS-Community/slimserver#677] [Q&A] [answered]
I changed the root password a decade ago on my SB Radio using ssh.
How can I reset it ?

**Comment by michaelherger** [comment, marked as answer]
I fear you'll have to factory reset the unit. Hold down Add ("+") and plug it in. Keep button held down until you see the "Free Your Music" screen.

**Reply by faxm0dem** [reply]
thanks, I'll try that !
